module/input/SensorFilter/src/SensorFilter.hpp

```diff
@@ -96,9 +96,31 @@ namespace module::input {
             } buttons;
 
             struct FootDown {
-                FootDown() : fromLoad(true), certaintyThreshold(0.05) {}
-                bool fromLoad;
-                float certaintyThreshold;
+                FootDown()
+                    : current_method("Z_HEIGHT")
+                    , certainty_thresholds({
+                          {"Z_HEIGHT", 0.01f},
+                          {"VIRTUAL", 0.05f},
+                          {"FSR", 60.0f},
+                      }) {}
+                FootDown(const std::string& method, const std::map<std::string, float>& thresholds) {
+                    set_method(method, thresholds);
+                }
+                void set_method(const std::string& method, const std::map<std::string, float>& thresholds) {
```

Comment:
Do we use this anywhere outside the constructor above it? If not, probably remove it? idk. Seems out of place

Reply:
We do, in `on<Configuration>`